fenix/app/src/main/java/org/mozilla/fenix/addons/ui/AddonPermissionsScreen.kt

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.addons.ui
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.LinkText
+import org.mozilla.fenix.compose.LinkTextState
+import org.mozilla.fenix.theme.FirefoxTheme
+
+private const val LEARN_MORE_URL =
+    "https://support.mozilla.org/kb/permission-request-messages-firefox-extensions"
```

Comment:
Let's use our sumo helper `SupportUtils.getSumoURLForTopic` make this site translatable you could find an example here https://github.com/mozilla-mobile/firefox-android/blob/3ce36aaabe426ddd40bc84646455c2a644fec5b4/fenix/app/src/main/java/org/mozilla/fenix/share/AddNewDeviceFragment.kt#L35

Reply:
@Amejia481 it looks like `permission-request-messages-firefox-extensions` does not link correctly when using the generated URL (ex: `https://support.mozilla.org/1/mobile/1.0.2413/Android/en-US/permission-request-messages-firefox-extensions`)

Who would we need to reach out to for getting this corrected? The current URL does redirect to the correct language in my experience but I agree it's correct to move this to the `getSumoURLForTopic()`

I've updated to the Sumo URL in the update [here](https://phabricator.services.mozilla.com/D205398) but we can't use it until it's worked out or reverted.

Reply:
Nevermind, I just missed the `getGenericSumoURLForTopic`. Updated 😄